apps/question-bank/src/test/java/gov/cdc/nbs/questionbank/page/content/staticelement/AddHyperLinkSteps.java

```diff
@@ -0,0 +1,67 @@
+package gov.cdc.nbs.questionbank.page.content.staticelement;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.AccessDeniedException;
+import org.springframework.security.authentication.AuthenticationCredentialsNotFoundException;
+import org.springframework.transaction.annotation.Transactional;
+import gov.cdc.nbs.questionbank.entity.WaTemplate;
+import gov.cdc.nbs.questionbank.entity.WaUiMetadata;
+import gov.cdc.nbs.questionbank.page.content.staticelement.request.PageStaticRequests;
+import gov.cdc.nbs.questionbank.page.content.tab.repository.WaUiMetaDataRepository;
+import gov.cdc.nbs.questionbank.support.ExceptionHolder;
+import gov.cdc.nbs.questionbank.support.PageMother;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+@Transactional
+public class AddHyperLinkSteps {
+    @Autowired
+    private PageStaticController pageStaticController;
+
+    @Autowired
+    private WaUiMetaDataRepository waUiMetaDataRepository;
+
+    @Autowired
+    private PageMother mother;
+
+    @Autowired
+    private ExceptionHolder exceptionHolder;
+
+    private Long hyperLinkId;
+
+    @When("I send a hyperlink request with {string} and {string}")
+    public void i_send_a_hyperlink_request(String label, String link) {
```

Comment:
todo: remind me to show you a really neat way to handle these types of interactions.  spoiler alert see `PagesResponseSteps`